modules/ROOT/Edits/x12-send-settings.adoc

```diff
@@ -0,0 +1,151 @@
+= Configure Send Settings for Outbound X12 Messages
```

Comment:
ditto edits to "Configure Send Settings for Outbound EDIFACT Messages" topic set.